### ISSUE TITLE
`katago runtests` fails in v1.12.0

### ISSUE THREAD

**Issue by p-linnane** [lightvector/KataGo#729]
Testing on macOS to package for Homebrew. Seeing this [in CI](https://github.com/Homebrew/homebrew-core/actions/runs/3865736735/jobs/6589449002) as well as locally.

```
katago runtests
Running rng and hash tests
Running fancy math tests
Running base64 tests
Running thread tests
Running board IO tests
Running board basic tests
Running board area tests
Running rules tests
Running board undo test
Running board handicap test
Running board stress test
Running sgf tests
Running basic symmetries tests
Running board symmetry tests
Running board replay test
Running config tests
libc++abi: terminating with uncaught exception of type IOError: Could not open file tests/data/configs/analysis_example.cfg - does not exist or invalid permissions?
[1]    60272 abort      katago runtests
```

**Comment by lightvector** [comment]
Mmm, I guess some recent tests there were unfortunately added under the assumption that KataGo would be run in the presence of the files in the git repo. Is that awkward to manage, should I push a change that disables those tests when the directory doesn't exist or something like that?

**Comment by p-linnane** [comment]
With Homebrew we simply try to run a basic functionality test after building from source. If you can provide some guidance on another small command we could run to test the application works we can use that instead. Don't want to make a bunch of work for you 😄

**Comment by lightvector** [comment]
Heh, thanks. But anyways, since I had to release a fix v1.12.1 for TensorRT users, I also included a disabling of that test in that new release. Can you try if it works now?

**Comment by p-linnane** [comment]
Confirming that v.1.12.1 fixed the issue and we are good to go. Thank you again!
